metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -4,6 +4,7 @@
 from dataclasses import dataclass, field
 from typing import Any, Dict, Iterable, List, Optional, Tuple, Type, cast
 
+import boto3
```

Comment:
We should add https://github.com/Affirm/datahub/blob/3e0fef1b7fd2beb6591a89beb644e63e73bc8abe/metadata-ingestion/setup.py#L91 to dbt's dependencies.

https://github.com/Affirm/datahub/blob/3e0fef1b7fd2beb6591a89beb644e63e73bc8abe/metadata-ingestion/setup.py#L180